src/screens/GroupInfo/index.js

```diff
@@ -27,6 +27,7 @@ import ModalActionAnonymous from './elements/ModalActionAnonymous';
 import ReportGroup from '../../assets/images/report.png';
 import useGroupInfo from './hooks/useGroupInfo';
 import {CHANNEL_TYPE_ANONYMOUS} from '../../utils/constants';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You have imported both `COLORS` from `'../../utils/theme'` and `colors` from `'../../utils/colors'`. If these two imports are serving the same purpose, it would be better to stick with one for consistency. If they serve different purposes, consider renaming one of them to avoid confusion.
```diff
- import {COLORS} from '../../utils/theme';
- import {colors} from '../../utils/colors';
+ import {themeColors} from '../../utils/theme';
+ import {utilityColors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->